src/KnownFile.cpp

```diff
@@ -1153,7 +1153,7 @@ void CKnownFile::CreateOfferedFilePacket(
 					tags.push_back(new CTagInt32(FT_FILESIZE_HI, (uint32)(GetFileSize() >> 32)));
 				}
 			} else {
-				if (!pClient->SupportsLargeFiles()) {
+				if (pClient && (!pClient->SupportsLargeFiles())) {
```

Comment:
The `wxASSERT` call on line 1093 properly checks for both `pClient` and `pServer` being non-null. However, only in debug builds (wxDEBUG enabled). That should be changed to something that works also in release builds (like `wxCHECK_RET`) and further checks can be omitted.